### ISSUE TITLE
Drop Enterprise Architect

### ISSUE THREAD

**Issue by ascheman** [aim42/htmlSanityCheck#326]
-

**Comment by gernotstarke** [comment]
Idea: replace either with draw.io diagrams (for static diagrams, like building-block view) or PlantUML (for dynamic diagrams, like sequence diagrams).

**Comment by ascheman** [comment]
Yes, and we could even use some LLM for the first conversions. ChatGPT created the following PlantUML 


![hsc-context.puml](//www.plantuml.com/plantuml/png/TP3HJi8m68Nl-nH_k32G00zWo48qHapm1lJEczeTvhAs8_t_HOdntTqwoB32tVRjV3qxNRBe88spIXkf0xqoml5nTDUKDgVjWGNkpOynB-PJUWmwjvYIqnKc8z9CCTQdb0emedrXGQELE4jRxKivqFq7J1NJ7UrXPFIjVlvOwzfeIqzE5-0eHD2_NmclNfxNj2ijnZmPb4XJuU5B47phVEATIm7J3GLmtGHph3ibDZqzbNRtGhDPblw_Ebj_yRjAG0oTRxebqt6-6enBXjY0_fyKi4E0D-3BlUcuePqMYqt9LPObvoT6qj9Q-vfbNtlahRg4pnjdVm40)

from

![hsc-context.png](https://github.com/aim42/htmlSanityCheck/blob/develop/src/docs/images/ea/htmlSanityCheck/hsc-context.png)

This is at least a start. Any recommendations, @rdmueller?

```plantuml
@startuml
actor User
actor "Build System\n(e.g., Gradle, make)" as BuildSystem

rectangle "Html Sanity Check" as HtmlSanityCheck {
}

rectangle "Local Images" as LocalImages {
}

rectangle "Local HTML file(s)" as LocalHTML {
}

rectangle "External websites & resources" as ExternalResources {
}

User --> HtmlSanityCheck
BuildSystem --> HtmlSanityCheck

HtmlSanityCheck --> LocalImages : checks
HtmlSanityCheck --> LocalHTML : checks
LocalHTML --> LocalImages : references

HtmlSanityCheck --> ExternalResources : checks <<Risk>>
LocalHTML --> ExternalResources : references <<postponed>>

@enduml
```

**Comment by rdmueller** [comment]
ChatGPT can create both, draw.io and plantUML diagrams. I can give you access to my frontend which is able to display plantUML directly via kroki and correct the code if it is broken. But you need an API key.

**Comment by ascheman** [comment]
Incorporated to `develop` branch
